cli/src/ai/analyze.ts

```
@@ -0,0 +1,86 @@
+import { readFile, writeFile, stat } from 'node:fs/promises'
```

ESLint

[error] Expected "stat" to come before "writeFile".

(perfectionist/sort-named-imports)